### ISSUE TITLE
Accessibility: JAWS reads "has flows from" after each element in extension Webview

### ISSUE THREAD

**Issue by RussellSprouts** [microsoft/vscode#79250]
- VSCode Version: 1.37.1 (user setup), 1.38.0-insider (user setup)
- OS Version: Windows 10.0.18362 Build 18362

JAWS version: JAWS 2019.1907.42

Steps to Reproduce:

1. Launch the JAWS screen reader
2. Navigate to a Webview panel. I can confirm the bug happens with the Webview created by "Release notes", markdown preview, and an extension
3. Use the arrow keys or tab to navigate in the webview.
4. JAWS will unnecessarily read "has flows from" after each element in the Webview.

I have created a minimal sample extension to see the issue and test it:
https://github.com/RussellSprouts/vscode-a11y-bugs-demo

Does this issue occur when all extensions are disabled?: YES

**Comment by RussellSprouts** [comment]
See also: https://github.com/FreedomScientific/VFO-standards-support/issues/242

**Comment by isidorn** [comment]
@RussellSprouts thanks for filing this issue. However we are aware of some JAWS limitations in VS Code.
For VS Code accessiblity we primarly test NVDA since it is open source.
Can you reproduce your issue with NVDA?

**Comment by RussellSprouts** [comment]
Thanks @isidorn. NVDA does not have the same issue.

**Comment by jooyoungseo** [comment]
@meganrogge and @rperez030 -- I think this is a long-standing and non-trivial issue for JAWS users. Is there any way for us to communicate with Vispero to address this issue?

**Comment by isidorn** [comment]
I know @rperez030 has some contacts there - Rob could you drive this?

**Comment by rperez030** [comment]
> Rob could you drive this?

absolutely. we have regular syncs with them, but so far, I cannot reproduce the issue. @jooyoungseo  I can use your help here. I'm running JAWS 2024. If I explore either the extension page for the WSL extension, for example, or the vS Code welcome page, using virtual pc cursor in both instances, either with arrow keys navigation or the tab key, I don't get the "has flows" message, either by speech or braille. I tried with default settings just in case. what could I be missing here?

**Comment by jooyoungseo** [comment]
@rperez030 -- If you open "Show Release Notes" under "Help" menu with JAWS and navigate the note with the virtual cursor, you would hear "has flows from" message.
